addons/portal/static/src/chatter/frontend/persona_model_patch.js

```diff
@@ -0,0 +1,12 @@
+import { Persona } from "@mail/core/common/persona_model";
+
+import { patch } from "@web/core/utils/patch";
+
+patch(Persona.prototype, {
+    get avatarUrl() {
+        if (this.store.env.services["portal.chatter"].portalSecurity?.token) {
+            return `/web/image/res.partner/${this.id}/avatar_128`;
+        }
```

Comment:
I don't get it. Checking existing of token to use standard route?
This doesn't look correct.
Also it's missing using write_date for caching.